### ISSUE TITLE
How to bind to specific interface ?

### ISSUE THREAD

**Issue by Sfinx** [Arriven/db1000n#520]
Having several interfaces and need to force usage of specific one. How to do this ? There is must be option "-I" like ping has or so.

**Comment by Arriven** [comment]
As of 0.8.32 you can specify `--local-address <ip>` to achieve that

**Comment by Sfinx** [comment]
Thanks !

**Comment by deputinizer** [comment]
Also fixed in a7edd0071ee129712fe3ebdf643a75e24c3f1a0a

**Comment by Sfinx** [comment]
Works with ipv6 ! But still small bug that was not present at pre 0.8x series - countrycheker still uses ipv4 addressing for country check :

```
utils/countrychecker.go:78      location info   {"country": ....
```
